### PR TITLE
APEXCORE-409 #resolve added documentation for app specification using json

### PR DESCRIPTION
**Pull request by davidyan74**
@PramodSSImmaneni  please review and merge

### CODE REVIEW

**Comment by pramodin** [comment]
Shouldn't this be in application developer guide instead of application packages

**Comment by davidyan74** [comment]
@PramodSSImmaneni  I thought about that too, but I think it's more useful for the user to learn about the JSON format in the context of app package.  Or we can have it in both places but I'm not sure whether it's a good idea to duplicate information

**Comment by pramodin** [comment]
I think we can keep in app dev guide because we also have the property way of building apps there (apart from java). You can link from app packages to that section.

**Comment by davidyan74** [comment]
@PramodSSImmaneni okay, please review again

File: docs/application_development.md [hunk 1]
```diff
@@ -1196,12 +1194,54 @@ public class Application implements StreamingApplication
 }
 ```
 
+JSON File DAG Specification
+--------------------------
+In addition to Java, you can also specify the DAG using JSON, provided the operators in the DAG are present in the dependency jars. In the src/main/resources/app directory of the app package project, you can put your JSON files there. This is the specification of a JSON file that specifies an application.
```
Comment (pramodin) [inline docs/application_development.md]: Can you add a blurb to create the app folder if it does not exist

**Pull request merged**